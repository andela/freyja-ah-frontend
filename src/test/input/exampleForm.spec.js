import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ExampleForm from '../../components/inputs/exampleForm';

const setUp = (props = {}) => {
  const component = shallow(
    <ExampleForm
      {...props}
      type="text"
      placeholder="Last name"
      name="lastName"
    />,
  );
  return component;
};
describe('ExampleForm component', () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  it('should render without errors', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
