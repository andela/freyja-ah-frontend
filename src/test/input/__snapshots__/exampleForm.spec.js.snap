// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleForm component should render without errors 1`] = `
<Fragment>
  <form
    className="example"
  >
    <InputField
      name="lastName"
      placeholder="Last name"
      type="text"
    />
  </form>
</Fragment>
`;
